drivers/mrf24j40/include/mrf24j40_registers.h

```diff
@@ -268,6 +268,14 @@ extern "C" {
 /** @} */
 
 /**
+ * @name    Shift offsets for TXSTAT register (0x24)
+ * @{
+ */
+#define MRF24J40_TXSTAT_MAX_FRAME_RETRIES_SHIFT 6
+#define MRF24J40_TXSTAT_CCAFAIL_SHIFT           5
```

Comment:
not used (yet?)

Reply:
Correct, I added it for completeness of that register definition

Reply:
fine by me, just asking.